site/v5/professional/professional.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Mongock Professional  
+date: 2014-04-18 11:30:00 
+permalink: /v5/professional/index.html
+eleventyNavigation:
+  version: v5
+  root: true
+  order: 80
+---
+<h1 class="title">Professional- Introduction</h1>
+
+[[TOC]]
+
+# What is Mongock professional?
+Mongock professional is the extension of the Mongock open source project, providing advanced features and support. 
+
+Professional support not only guarantees that your business will get all the problems and questions solved, but also that the project is properly mantained and not only by the community, but there is also a company behind it, taking care of it.
+
+# What does the professional distribution provide?
+
+As mentioned, in summary Mongock professional provides business support(with different levels depending on the business needs) and advanced features. This features are mainly focussed in the following areas:
+
+- Management
+- Deployment
+- CLI
+- Dashboard
+- Centralised migration server
+- Other advanced features like multi-tenant, etc.
```

Comment:
Given this is the official docs - we should be specific (not put etc.)

Reply:
I just removed the entire list